test/parallel/test-fs-read-buffer-zero-length.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+var common = require('../common');
+var assert = require('assert');
+var path = require('path'),
```

Comment:
You guys should really add an `eslint` rule for this. So much overhead for contributors to deal with the back-and-forth.